cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java

```diff
@@ -241,7 +241,11 @@ protected VaultConfig readVaultConfig(final Session<?> session) throws Backgroun
         }
     }
 
-    private VaultConfig readVaultConfig(final String token) {
+    protected String readVaultConfig(final Session<?> session) throws BackgroundException {
+        return new ContentReader(session).read(config);
+    }
+
+    public static VaultConfig readVaultConfigFromJWT(final String token) {
```

Comment:
Rename to `parseVaultConfigFromJWT`

Reply:
@dkocher Final? https://github.com/iterate-ch/cyberduck/pull/14715#discussion_r1205580994

Reply:
done.